_docs/performance-tuning/drill-metastore/030-drill-iceberg-metastore.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Drill Iceberg Metastore"
+parent: "Drill Metastore"
+date:
+---
+
+Drill uses Iceberg Metastore implementation based on [Iceberg tables](http://iceberg.incubator.apache.org). For
```

Comment:
Please add:
For Drill 1.17, this is default Drill Metastore implementation.

Reply:
Thanks, added.